tequilapi/endpoints/pilvytis.go

```diff
@@ -202,11 +203,37 @@ func (e *pilvytisEndpoint) GetPaymentOrderCurrencies(w http.ResponseWriter, r *h
 	utils.WriteAsJSON(resp, w)
 }
 
+// GetPaymentOrderOptions returns recommendation on myst amounts
+//
+// swagger:operation GET /payment-order-options Order GetPaymentOrderOptions
+// ---
+// summary: Get payment order options
+// description: Includes minimum amount of myst required when topping up and suggested amounts
+// responses:
+//   200:
+//     description: PaymentOrderOptions object
+//     schema:
+//       "$ref": "#/definitions/PaymentOrderOptions"
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+func (e *pilvytisEndpoint) GetPaymentOrderOptions(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	resp, err := e.api.GetPaymentOrderOptions()
+	if err != nil {
+		utils.SendError(w, err, http.StatusInternalServerError)
+		return
+	}
+
+	utils.WriteAsJSON(resp, w)
```

Comment:
I think it's typical for tequila to returns objects which are described in the `contract` package.

While sometimes it can be repetitive it sort of protects us from leaking too much information as they act like "public" structs, not only that but they also keep things like  swagger generation in a single place and make developing the tequila `client` or using these endpoints in for example in `cli` a lot cleaner because we dont have to actually import every package in the whole project.

See this for example: https://github.com/mysteriumnetwork/node/pull/2943/files#diff-437c90b69c7d2e457bcc7e0afaacec573c4622bac9d913441ecedd8a323eacd3R176